web/rootfs/defaults/settings-config.js

```diff
@@ -406,30 +400,56 @@ config.disableDeepLinking = {{ $DISABLE_DEEP_LINKING }};
 config.p2p.preferredCodec = '{{ .Env.P2P_PREFERRED_CODEC }}';
 {{ end -}}
 
-// Enable preferred video Codec
-if (!config.hasOwnProperty('videoQuality')) config.videoQuality = {};
+// Video quality settings.
+//
+
+config.videoQuality = {};
 {{ if .Env.VIDEOQUALITY_PREFERRED_CODEC -}}
 config.videoQuality.preferredCodec = '{{ .Env.VIDEOQUALITY_PREFERRED_CODEC }}';
 {{ end -}}
 
-config.videoQuality.enforcePreferredCodec = {{ $VIDEOQUALITY_ENFORCE_PREFERRED_CODEC }};
-
-if (!config.videoQuality.hasOwnProperty('maxBitratesVideo')) config.videoQuality.maxBitratesVideo = null;
-{{ if and .Env.VIDEOQUALITY_BITRATE_H264_LOW .Env.VIDEOQUALITY_BITRATE_H264_STANDARD .Env.VIDEOQUALITY_BITRATE_H264_HIGH -}}
-config.videoQuality.maxBitratesVideo = config.videoQuality.maxBitratesVideo || {}
-config.videoQuality.maxBitratesVideo.H264 = { low: {{ .Env.VIDEOQUALITY_BITRATE_H264_LOW }}, standard: {{ .Env.VIDEOQUALITY_BITRATE_H264_STANDARD }}, high: {{ .Env.VIDEOQUALITY_BITRATE_H264_HIGH }} };
+{{ if and .Env.VIDEOQUALITY_BITRATE_AV1_LOW .Env.VIDEOQUALITY_BITRATE_AV1_STANDARD .Env.VIDEOQUALITY_BITRATE_AV1_HIGH .Env.VIDEOQUALITY_BITRATE_AV1_SS_HIGH -}}
```

Comment:
So, it's necessary to set them all even if one wants to bump just the high?

Reply:
The code doesn't dictate it but do you see value in giving the option to bump only one of the values? Typically, when someone changes the high, I'd expect them to change standard and low accordingly.

Reply:
I don't know, just asking :-) I guess we'll find out if someone needs it eventually.